### PR TITLE
fillerama api seems to be back up.

### PR DESCRIPTION
**Pull request by mnr**
## Description
fillerama API seems to work fine, so I uncommented the related code. As a precaution, I've wrapped the jsonlite call with try() to trap out api errors.

## Example
say(what="futurama") # this should work.

### CODE REVIEW

**Comment by sckott** [comment]
Thanks @mnr 

@aedobbyn Thoughts on this? I'm thinking maybe we don't rely on this API that could go down again probably pretty easily.

**Comment by mnr** [comment]
I've been pointing to cowsay in a couple of [R weekly sessions](https://www.linkedin.com/learning/r-for-data-science-lunchbreak-lessons/cowsay). One of my students pointed out the missing "lorem ipsum" - no big deal if this is removed - but just to let you know...

**Comment by mnr** [comment]
oops - didn't mean to close the pull

**Comment by aedobbyn** [comment]
thanks @mnr!

> I'm thinking maybe we don't rely on this API that could go down again probably pretty easily.

hmm yeah... how often has it gone down? I'm not at all attached to it so my instinct would also be to just remove, but if we want to keep it in maybe we handle errors with a `tryCatch` or a `purrr::possibly`?

**Comment by mnr** [comment]
I put in a simple try(,,) but only to keep it from crashing out - nothing graceful.

Would you like me to add a more interesting handler if the API isn't available? I'd be happy to close this pull, add something more elegant, then re-pull....

**Comment by aedobbyn** [comment]
Yeah I think something that lets the user know what's going on if the API is down makes more sense. Either that or we take this out entirely. Up to you @sckott

**Comment by sckott** [comment]
Okay, we can keep it. 

However, there needs to be better error handling. e.g, 

```r
what <- try(jsonlite::fromJSON(paste0('http://api.chrisvalleskey.com/fillerama/get.php?count=1&format=json&show=', what))$db$quote )
if (inherits(what, "try-error")) stop(what)
```

haven't tested that, but something like that

If code is left as is right now, then on an error it would just return futurama as the string in the output because `what` would retain its original value as passed in by the user

**Comment by mnr** [comment]
I've added filleramaIsBroken to substitute something when the fillerama api is broken. I would LOVE to return something appropriate to "what" but can't seem to figure out how to pass that variable to filleramaIsBroken. Please let me know if I'm missing something obvious.

**Comment by aedobbyn** [comment]
If I'm understanding what you want to do, you could do something like

```
if (what %in% c("arresteddevelopment", "doctorwho", "dexter", "futurama", "holygrail", "simpsons", "starwars", "loremipsum")) {
    what_orig <- what

    check4jsonlite()
    what <- tryCatch(
      jsonlite::fromJSON(paste0("http://api.chrisvalleskey.com/fillerama/get.php?count=1&format=json&show=", what))$db$quote,
      error =
        function(e) {
          return(paste0("`", what_orig, "` isn't available right now :("))
        }
    )
  }
```

so that would return

```
> say(what = "starwars")

 -------------- 
`starwars` isn't available right now :( 
 --------------
    \
      \
        \
            |\___/|
          ==) ^Y^ (==
            \  ^  /
             )=*=(
            /     \
            |     |
           /| | | |\
           \| | |_|/\
      jgs  //_// ___/
               \_)
```

Or if you want it to say something besides just `what_orig` you could make a key-value dataframe for the phrase you want it to say given a `what_orig` if that makes sense

**Comment by sckott** [comment]
nice one @aedobbyn , looks good

**Comment by mnr** [comment]
Thanks @aedobbyn ! 

I was trying to incorporate a separate function to keep all the awkward error handling elsewhere. But Amanda's solution works great.

**Comment by aedobbyn** [comment]
cool! you could definitely pull that into a function if you wanted to. up to you!

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ cowsay
 If you are familiar with `cowsay` on the cli, then you know what this
 is, but for R. If not, read below. Why? Why not?
 
-### Contributors (alphebetical)
```
Comment (sckott) [inline README.md:14]: @mnr please undo this change - although it needs to be fixed, PR's should stay focused on one issue instead of pulling in other fixes

File: R/utils.R [hunk 2]
```diff
@@ -19,3 +19,13 @@ check_color <- function(clr) {
   }
   return(clr)
 }
+
+check4pkg <- function(pkg) {
```
Comment (sckott) [inline R/utils.R:23]: @mnr please undo this change (i.e., put check4pkg back into zzz.R) - although perhaps these helper fxns should be in the same file, the change is completely unrelated to the focus of your PR

**Review by sckott — CHANGES_REQUESTED** [review]
a couple more change requests, thanks!

File: README.md (hunk 1 above)
Comment (mnr) [reply README.md:14]: agreed and done. Revised pull to come soon...

File: R/utils.R (hunk 2 above)
Comment (mnr) [reply R/utils.R:23]: This was an early change in the PR. It will be easier to reset this entire pull request and restart

**Comment by mnr** [comment]
let me clean up and resubmit...

File: R/utils.R (hunk 2 above)
Comment (sckott) [reply R/utils.R:23]: okay, sounds good

Comment (mnr) [reply R/utils.R:23]: I'll stand by on this until you have a chance to merge/reject the typo. Then I'll pull down a clean copy and rebuild the try-catch.
